docs/LwM2M_Client/Nordic/Runtime_certificate.md

```diff
@@ -0,0 +1,107 @@
+
+## Prerequisites
+* The nRF9160DK board with a USB cable.
+* Cloned [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client#getting-started) repository
+* Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+* Installed [OpenSSL](https://www.openssl.org/source/)
+* Installed **minicom** (for Linux), **RealTerm**, **PuTTy** (for Windows), or another serial communication program.
+* An active [Coiote IoT DM](https://eu.iot.avsystem.cloud/) user account.
+
+!!! Note
+    In this tutorial we will use the nRF9160DK board as an example.
+
+## Build and flash the device
+0. Connect the nRF9160DK board to a USB port of your machine.
+0. Go to your local **Anjay-zephyr-client/demo** directory
+0. Set manifest file to `west-nrf.yml`, and do `west update`:
+
+    ```
+    west config manifest.file west-nrf.yml
+    west update
+    ```
+
+0. Build a project with a runtime certificate and private key.
+
+    ```
+    west build -b <BOARD> -p -- -DCONFIG_ANJAY_ZEPHYR_RUNTIME_CERT_CONFIG=y
+    ```
+
+    where <BOARD\> should be replaced by the selected board from `boards/` directory.
+
+    !!! Note
+
+        The runtime certificate and private key configuration do not work with Thingy:91. Other boards should work with this command.
+
+    This feature works with nRF9160DK starting from revision v0.14.0. For this board use configuration that utilizes an external flash chip and software-based cryptography:
+
+    ```
+    west build -b nrf9160dk_nrf9160_ns@0.14.0 -p -- -DCONF_FILE=prj_extflash.conf -DOVERLAY_CONFIG="overlay_nrf_mbedtls.conf"
+    ```
+
+0. Flash the board with `west flash` command.
+
+## Generate certificate
+The certificate and private key based on the SECP256R1 curve can be provided through the shell interface in `.pem` format. To generate them open terminal in the **Anjay-zephyr-client/demo** directory and use the following commands.
+
+```
+openssl ecparam -name secp256r1 -out ecparam.der
+openssl req -new -x509 -nodes -newkey ec:ecparam.der -keyout demo-cert.key -out demo-cert.crt -days 3650
+openssl x509 -in demo-cert.crt -outform pem -out cert.pem
+openssl ec -in demo-cert.key -outform pem -out key.pem
+```
+
+![Fragment of creating certificates](images/create_cert.png)
+
+!!! Important
+
+    To use the certificate and private key with Coiote IoT DM you must specify a common name that is the same as the client endpoint name.
+
+You will see created `cert.pem` and `key.pem` files in the `Anjay-zephyr-client/demo` directory.
+
+## Configure the Client
+0. Provide the generated certificate and private key through the shell in the serial communication program.
+
+    !!! Note
+
+        Before setting configurations in Anjay you need to stop running Anjay by `anjay stop` command.
+
+    ```
+    anjay config set public_cert
+    anjay config set private_key
```

Comment:
Probably it would be nice to say call this command and paste the content of the .pem file or something.